package/yast2-geo-cluster.spec

```diff
@@ -12,30 +12,29 @@
 # license that conforms to the Open Source Definition (Version 1.9)
 # published by the Open Source Initiative.
 
-# Please submit bugfixes or comments via http://bugs.opensuse.org/
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
 #
 
 
 Name:           yast2-geo-cluster
-Version:        4.2.0
+Version:        4.2.1
 Release:        0
+Summary:        Configuration of booth
+License:        GPL-2.0-only
+Group:          System/YaST
+Url:            https://github.com/yast/yast-geo-cluster
 
-BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 Source0:        %{name}-%{version}.tar.bz2
 
 BuildRequires:  update-desktop-files
 BuildRequires:  yast2-devtools >= 3.1.10
```

Comment:
%yast_metainfo is added in 4.2.2, so dependencies of yast2-devtools should update.